slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py

```diff
@@ -0,0 +1,611 @@
+            extreme_points, result_file = logic.segmentation(model, inputVolume)
+            if self.updateSegmentationMask(extreme_points, result_file, modelInfo):
+                result = 'SUCCESS'
+                self.segmentMarkupNode.RemoveAllMarkups()
+                self.updateGUIFromMRML()
+        except:
+            qt.QApplication.restoreOverrideCursor()
+            progressBar.close()
+            slicer.util.errorDisplay(operationDescription + " - unexpected error.", detailedText=traceback.format_exc())
+            return
+
+        qt.QApplication.restoreOverrideCursor()
+        progressBar.close()
+
+        logging.info("++ Time consumed by onClickSegmentation: {}".format(time.time() - start))
+        msg = 'Run segmentation for ({}): {}\t\nTime Consumed: {} (sec)'.format(model, result, (time.time() - start))
+        logging.info(msg)
+        qt.QMessageBox.information(slicer.util.mainWindow(), 'NVIDIA AIAA', msg)
+
+        self.onClickEditPoints()
+
+    def getFiducialPointsXYZ(self):
+        v = self.scriptedEffect.parameterSetNode().GetMasterVolumeNode()
+        RasToIjkMatrix = vtk.vtkMatrix4x4()
+        v.GetRASToIJKMatrix(RasToIjkMatrix)
+
+        point_set = []
+        n = self.segmentMarkupNode.GetNumberOfFiducials()
+        for i in range(n):
+            coord = [0.0, 0.0, 0.0]
+            self.segmentMarkupNode.GetNthFiducialPosition(i, coord)
+
+            p_Ras = [coord[0], coord[1], coord[2], 1.0]
+            p_Ijk = RasToIjkMatrix.MultiplyDoublePoint(p_Ras)
+
+            logging.info('From Fiducial: {} => {}'.format(coord, p_Ijk))
+            point_set.append(p_Ijk[0:3])
+
+        logging.info('Current Fiducials-Points: {}'.format(point_set))
+        return point_set
+
+    def onClickAnnotation(self):
+        start = time.time()
+        model = self.annotationModelSelector.currentText
+        label = self.getActiveLabel()
+        operationDescription = 'Run Annotation for model: {} for label: {}'.format(model, label)
+        logging.info(operationDescription)
+        progressBar = slicer.util.createProgressDialog(windowTitle="Wait...", labelText=operationDescription,
+                                                       maximum=100)
+        slicer.app.processEvents()
+        try:
+            qt.QApplication.setOverrideCursor(qt.Qt.WaitCursor)
+            logic = AIAALogic(self.serverUrl.text,
+                              progress_callback=lambda progressPercentage,
+                                                       progressBar=progressBar: SegmentEditorEffect.report_progress(
+                                  progressBar, progressPercentage))
+
+            inputVolume = self.scriptedEffect.parameterSetNode().GetMasterVolumeNode()
+            modelInfo = self.models.get(model)
+            pointSet = self.getFiducialPointsXYZ()
+
+            result_file = logic.dextr3d(model, pointSet, inputVolume, modelInfo)
+            result = 'FAILED'
```

Comment:
is this variable `result` necessary?
or just a literal is enough
cuz we are not using it if something actually failed

Reply:
It's just for the logging/message